tutorials/plot_artifacts_correction_ica.py

```diff
@@ -160,12 +169,16 @@
 # What if we don't have an EOG channel?
 # -------------------------------------
 #
+# We could either:
+#
 # 1) make a bipolar reference from frontal EEG sensors and use as virtual EOG
 # channel. This can be tricky though as you can only hope that the frontal
 # EEG channels only reflect EOG and not brain dynamics in the prefrontal
 # cortex.
-# 2) Go for a semi-automated approach, using template matching.
-# In MNE-Python option 2 is easily achievable and it might be better,
+#
+# 2) go for a semi-automated approach, using template matching.
```

Comment:
These are interpreted as ReStructuredText, so you need to [follow their standard](http://docutils.sourceforge.net/docs/user/rst/quickref.html#enumerated-lists):
```
1. blah blah
   blah blah blah
2. blah blah ...
```